helm/metrics-server-app/templates/ciliumnetworkpolicy.yaml

```diff
@@ -0,0 +1,17 @@
+{{ if .Values.ciliumNetworkPolicy.enabled }}
```

Comment:
Can you add the capabilities check?

Reply:
we don't want that!
If you add the capabilities check, if this app is installed before cilium in a cluster then installation will succeed, but the cilium network policy will NOT be created, meaning metrics-server wont work.
If the `ciliumNetworkPolicy.enabled` then helm chart creation fails if cilium is not yet installed and that is exactly the desired behavior